<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="5" hostname="Sudarshan" failures="1" tests="2" name="Default test(failed)" time="0.354" errors="0" timestamp="2022-03-22T22:38:08 IST">
  <testcase classname="failedxml.FailedTestCases_study" name="test6" time="0.241">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method failedxml.FailedTestCases_study.test6() didn&amp;apos;t finish within the time-out 200">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method failedxml.FailedTestCases_study.test6() didn't finish within the time-out 200
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- test6 -->
  <testcase classname="failedxml.FailedTestCases_study" name="test7" time="0.01"/>
  <testcase classname="failedxml.FailedTestCases_study" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
  <testcase classname="failedxml.FailedTestCases_study" name="test4" time="0.0">
    <ignored/>
  </testcase> <!-- test4 -->
  <testcase classname="failedxml.FailedTestCases_study" name="test5" time="0.0">
    <ignored/>
  </testcase> <!-- test5 -->
  <testcase classname="failedxml.FailedTestCases_study" name="test2" time="0.0">
    <ignored/>
  </testcase> <!-- test2 -->
  <testcase classname="failedxml.FailedTestCases_study" name="test3" time="0.0">
    <ignored/>
  </testcase> <!-- test3 -->
</testsuite> <!-- Default test(failed) -->
